doc/make_numpy_coverage_table.py

```diff
@@ -0,0 +1,247 @@
+            raise ValueError(f"'{name}' not found in numpy namespace")
+
+        in_context = {}
+        for ctx in contexts:
+            try:
+                _ = getattr(ctx.np, name)
+            except AttributeError:
+                in_context[type(ctx)] = "No"
+
+        numpy_functions_for_context[name] = (directive, in_context)
+
+    return numpy_functions_for_context
+
+# }}}
+
+
+# {{{ writing
+
+def write_array_creation_routines(outf, contexts):
+    # https://numpy.org/doc/stable/reference/routines.array-creation.html
+    funcs = (
+            # (sphinx-directive, name)
+            ("func", "empty_like"),
+            ("func", "ones_like"),
+            ("func", "zeros_like"),
+            ("func", "full_like"),
+            ("func", "copy"),
+            )
+
+    r = TABLE_TEMPLATE.render(
+            title="Array creation routines",
+            contexts=contexts,
+            numpy_functions_for_context=build_supported_functions(funcs, contexts),
+            )
+    outf.write(r)
+
+
+def write_array_manipulation_routines(outf, contexts):
+    # https://numpy.org/doc/stable/reference/routines.array-manipulation.html
+    funcs = (
+            # (sphinx-directive, name)
+            ("func", "reshape"),
+            ("func", "ravel"),
+            ("func", "transpose"),
+            ("func", "broadcast_to"),
+            ("func", "concatenate"),
+            ("func", "stack"),
+            )
+
+    r = TABLE_TEMPLATE.render(
+            title="Array manipulation routines",
+            contexts=contexts,
+            numpy_functions_for_context=build_supported_functions(funcs, contexts),
+            )
+    outf.write(r)
+
+
+def write_linear_algebra(outf, contexts):
+    # https://numpy.org/doc/stable/reference/routines.linalg.html
+    funcs = (
+            # (sphinx-directive, name)
+            ("func", "vdot"),
+            )
+
+    r = TABLE_TEMPLATE.render(
+            title="Linear algebra",
+            contexts=contexts,
+            numpy_functions_for_context=build_supported_functions(funcs, contexts),
+            )
+    outf.write(r)
+
+
+def write_logic_functions(outf, contexts):
+    # https://numpy.org/doc/stable/reference/routines.logic.html
+    funcs = (
+            # (sphinx-directive, name)
+            ("func", "all"),
+            ("func", "any"),
+            ("data", "greater"),
+            ("data", "greater_equal"),
+            ("data", "less"),
+            ("data", "less_equal"),
+            ("data", "equal"),
+            ("data", "not_equal"),
+            )
+
+    r = TABLE_TEMPLATE.render(
+            title="Logic Functions",
+            contexts=contexts,
+            numpy_functions_for_context=build_supported_functions(funcs, contexts),
+            )
+    outf.write(r)
+
+
+def write_mathematical_functions(outf, contexts):
+    # https://numpy.org/doc/stable/reference/routines.math.html
+    funcs = (
+            ("data", "sin"),
+            ("data", "cos"),
```

Comment:
A lot of the functions listed in [_numpy_math_functions](https://github.com/inducer/arraycontext/blob/5218b2d6398d818f517d3a9856a9dabb6076dea0/arraycontext/fake_numpy.py#L78-L129) don't actually work. They error out with something like
```
loopy.diagnostic.LoopyError: could not determine type of 'out' of kernel 'actx_special_sinc'.
```

Reply:
Those are just not available in loopy then. Not ideal, but that's life for now.

Reply:
Wouldn't it be better to comment them out in `_numpy_math_functions` so that we get the nicer `AttributeError`? That `LoopyError` traceback is very scary!

Reply:
I'd rather improve the error message in loopy. If we comment them out, then there are two sources of truth on what's supported. Otherwise there's only one, which IMO is preferable.

Reply:
That does sound better, yeah.